File: Projeto-MobiFix/Database/dtos/userDtos/clienteDto.js
const paraClienteDto = (c) => {
    if (!c) return null;
    return {
        NIF: c._id.toString(),
        Nome: c.nome,
        Telefone: c.telefone,
        Email: c.email,
        Morada: c.morada || null,
        PasswordHash: c.passwordHash
    };
};

module.exports = { paraClienteDto };

File: Projeto-MobiFix/Backend/Models/Utilizadores/ClienteDto.cs
namespace Backend.Models;

public class ClienteDto
{
    public string NIF { get; set; } = string.Empty;
    public string Nome { get; set; } = string.Empty;
    public string Telefone { get; set; } = string.Empty;
    public string Email { get; set; } = string.Empty;
    public string Morada { get; set; } = string.Empty;
    public string PasswordHash { get; set; } = string.Empty;
}